Hymns/Display/DisplayHymnView.swift

```diff
@@ -27,9 +35,29 @@ struct DisplayHymnView: View {
                     } else {
                         viewModel.currentTab.content
                     }
+                    if !self.showSoundCloud && self.initiatedSoundCloud && AVAudioSession.sharedInstance().isOtherAudioPlaying {
```

Comment:
Is there a way to make it listen to when playback ends?

Reply:
I know... We need this. I wish it would flip our isOtherAudioPlaying variable between songs. Unfortunately, it just keeps playing right into the next song. I'm not sure if there is something fired on the website that we can listen for using JavaScript. That is what we would have to do I think.

Reply:
Okay, that's fine then